### ISSUE TITLE
Not aligned Properly

### ISSUE THREAD

**Issue by VaibhavSharma24** [PostHog/posthog#18255]
## Bug description

*I think its not aligned properly it is more towards left*  
*If this affects the front-end, screenshots would be of great help.*  

![Screenshot 2023-10-27 215159](https://github.com/PostHog/posthog/assets/72144747/5fe5e506-45b2-470d-94f0-552375674a49)

**Comment by shobhit8797** [comment]
According to my understanding the left alignment mentioned is for the images, right?
if I am wrong please let me know what is ` more towards left`

**Comment by VaibhavSharma24** [comment]
> According to my understanding the left alignment mentioned is for the images, right? if I am wrong please let me know what is ` more towards left`

Yes absolutely right its left alignment mentioned is for the images as well as content written below it as you can see they are not properly aligned to even its heading
